content/Gems You Want To Have In Your Rails API - Juan Diego Rosales.md

```diff
@@ -1,44 +1,56 @@
 # Gems You Want To Have In Your Rails API
 
-*Juan Diego Rosales is a software developer on the Rootstrap team and writes about his experience working on different projects. You can also follow Juan on [LinkedIn](www.linkedin.com/in/juan-diego-rosales-02a05a144).*
-
 Are you new to Rails? Or maybe just looking for any cool gem to use in your new API? Here is a list of gems we have on our Rails API Base that we consider a must-have in every Rails API project.
 
 ## Authentication
-If you are building a Rails API, then you will probably need token based authentication and that is when [Devise Token Auth](https://github.com/lynndylanhurley/devise_token_auth) comes in. This gem is built on top of Devise, one of the most popular authentication gems for Rails which you may have already worked with. If that is the case, then you'll get familiar with Devise Token Auth very easily.
+If you are building a Rails API, then you will probably need token-based authentication and that is when [Devise Token Auth](https://github.com/lynndylanhurley/devise_token_auth) comes in. This gem is built on top of Devise, one of the most popular authentication gems for Rails which you may have already worked with. If that's the case, then you'll get familiar with Devise Token Auth very easily.
 
 With very little setup, this gem provides registration, sign in, sign out and reset password flows out of the box, as well as very useful helper functions. Also, it handles email confirmation if enabled and supports the use of multiple user models, giving you the possibility to manage the authentication of different types of users at the same time using groups.
 
-In comparison to other alternatives available, one of the best perks that comes with Devise Token Auth, which is inherited from Devise, is the very little effort you have to invest to get things up and running. On top of that, you can also customize the different flows mentioned if you want to offer a different experience, and while this may be a little more challenging, it's not rocket science!
+In comparison to other alternatives available, one of the best perks that comes with Devise Token Auth is the very little effort you have to invest to get things up and running. On top of that, you can also customize the different flows mentioned if you want to offer a different experience, and while this may be a little more challenging, it's not rocket science!
+
+## Authorization
+I guess authorizing every request on an API can be achieved in the controller, but that is not scalable at all and may ease the introduction of vulnerabilities in an application. Here at Rootstrap, we find [Pundit](https://github.com/varvet/pundit) to be the perfect ally against those issues.
+
+Not only you will have one place in the codebase for the authorization logic, making the system more robust, but also it will help to clean up the controllers, improving the readability.
```

Comment:
I'm not 100% sure but I think it should be `Not only will you have`

Reply:
Agree